server/src/metadata/metadata.datasource.ts

```diff
@@ -27,6 +29,8 @@ export const typeORMMetadataModuleOptions: TypeOrmModuleOptions = {
     AlterFieldMetadataTable1695717691800,
     AddTargetColumnMap1696409050890,
     MetadataNameLabelRefactoring1697126636202,
+    RemoveFieldMetadataPlaceholder1697471445015,
```

Comment:
Not really sure why we changed this config? We now need to provide the name of each new migration in the config.